app/controllers/employee_invitation_controller.rb

```diff
@@ -0,0 +1,19 @@
+class EmployeeInvitationController < ApplicationController
+  before_action :set_company
+
+  def create
+    @email = params[:email]
```

Comment:
Do you use these instance variables in some view? (@email, @user)

Reply:
If I'm not wrong the company's name and user's name are being used in the view. Once it's true, I think the @email can be passed through `with` to keep it simple. WDYT @joaoGabriel55 ?

Reply:
I'm sorry guys I was confused, I thought we were talking about the mailer lol. @joaoGabriel55 there is no view for the controllers since this repo is only the backend side, and in controllers, we can easily track the variables in our specs, once they are instance variables